JamfSync/Model/Package.swift

```diff
@@ -9,9 +9,42 @@ struct Package: Identifiable {
     var jamfProId: Int?
     var displayName: String
     var fileName: String
-    var category: String
     var size: Int64?
     var checksums = Checksums()
+    var category: String?
+    var categoryId: String?
+    var info: String?
+    var notes: String?
+    var priority: Int?
+    var osRequirements: String?
+    var fillUserTemplate: Bool?
+    var indexed: Bool?     // Not to be updated
```

Comment:
consider making these private(set) so that they can't be updated outside the file?

Reply:
They need to be accessed from outside of this class, so I can't make them private, and they are also need to be set when they're loaded, so I can make them let's instead of vars. The comment just indicates that those fields shouldn't be sent to the server, only received.